core/src/main/java/com/linecorp/armeria/client/AggregatedHttpRequestHandler.java

```diff
@@ -95,4 +106,15 @@ void onWriteSuccess() {
     void cancel() {
         cancelled = true;
     }
+
+    @Override
+    void abort() {
+        assert aReq != null;
+        aReq.content().close();
+    }
+
+    @Override
+    void resume() {
+        writeDataOrTrailers();
```

Comment:
Although most paths are covered, I wasn't able to find a channel flush being done when writing http2 data/trailers.
What do you think of adding a `channel().flush()` at the end of this method?

```suggestion
        writeDataOrTrailers();
        channel().flush();
```